datalad/distribution/dataset.py

```diff
@@ -204,6 +206,47 @@ def get_subdatasets(self, pattern=None, fulfilled=None, absolute=False,
         else:
             return submodules
 
+    def create_subdataset(self, path, name=None,
+                          description=None,
+                          no_annex=False,
+                          annex_version=None,
+                          annex_backend='MD5E',
+                          git_opts=None,
+                          annex_opts=None,
+                          annex_init_opts=None):
+
+        # TODO: use `name` for subdatasets. (not necessarily equals `path`)
```

Comment:
We should add an argument for that right away, even if it doesn't do anything yet.